src/libsyntax/feature_gate/check.rs

```diff
@@ -150,48 +121,70 @@ impl<'a> PostExpansionVisitor<'a> {
             "sysv64" |
             "system" => {}
             "rust-intrinsic" => {
-                gate_feature_post!(&self, intrinsics, span,
-                                   "intrinsics are subject to change");
+                self.gate(
+                    span, sym::intrinsics, |f| f.intrinsics,
```

Comment:
I dislike this duplication. I'm assuming you want to fix it with the hashset you mentioned? I'd prefer it if this commit weren't in this PR but instead part of the hashset PR

Reply:
Yeah the duplication is gone with the hashset; I'll drop the commit for now.

Reply:
Dropped the commit.